selftests/unit/test_safeloader.py

```diff
@@ -398,6 +411,17 @@ def test_methods_order(self):
         self.assertEqual(expected_order, methods)
         avocado_keep_methods_order.remove()
 
+    def test_failed_discovery(self):
```

Comment:
@nicob87 FIY I've incorporated this code here: https://github.com/avocado-framework/avocado/pull/3382

Reply:
Thanks!